--- source\configure.orig	2017-02-03 16:09:07.685270300 +1000
+++ source\configure	2017-02-03 16:53:00.111505400 +1000
@@ -4357,7 +4357,7 @@
             *-*-cygwin)
                 if test "`$CC /help 2>&1 | head -c9`" = "Microsoft"
                 then
-                    CFLAGS="$CFLAGS /W4"
+                    CFLAGS="$CFLAGS"
                 fi ;;
             *-*-mingw*)
                 CFLAGS="$CFLAGS -W4" ;;
@@ -4371,7 +4371,7 @@
             *-*-cygwin)
                 if test "`$CXX /help 2>&1 | head -c9`" = "Microsoft"
                 then
-                    CXXFLAGS="$CXXFLAGS /W4"
+                    CXXFLAGS="$CXXFLAGS"
                 fi ;;
             *-*-mingw*)
                 CFLAGS="$CFLAGS -W4" ;;
